tests/conftest.py

```diff
@@ -19,6 +19,7 @@
 from tox.tox_env.python.virtual_env.api import VirtualEnv
 
 if TYPE_CHECKING:
+    from build._types import Distribution
```

Comment:
@layday: This is a private module, but its describing the API for a public interface. Maybe it should be more public?

Reply:
It is public: `from build import DistributionType`.

Reply:
Sorry, missed that because I was looking for the same name.